resources/playlist/playlist.models.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+const PlaylistSchema = Schema({
+  user_id: {
```

Comment:
Hi Joba, the playlist schema should just contain a string, an id, and maybe createdAt. You parse the string into the youtube API, and get all other information.

Reply:
Update the playlist to have an auto generate id, videoUrl, thumbnailUrl, and title.